src/tests/ftest/util/test_utils_pool.py

```diff
@@ -540,6 +540,18 @@ def drain(self, rank, tgt_idx=None):
         """
         return self.dmg.pool_drain(self.identifier, rank, tgt_idx)
 
+    @fail_on(CommandFailure)
+    def disable_aggregation(self):
+        """ Disable pool aggregation."""
+        self.log.info("Disable pool aggregation for pool %s", self.identifier)
```

Comment:
This will give us both the label (if set) and uuid:
```suggestion
        self.log.info("Disable pool aggregation for %s", str(self))
```

Reply:
Please see the update.